DNB.js

```diff
@@ -0,0 +1,290 @@
+		"items": [
+			{
+				"itemType": "book",
+				"title": "Ausgeplündert, zurückerstattet und entschädigt Arisierung und Wiedergutmachung in Mannheim",
+				"creators": [
+					{
+						"firstName": "Christiane",
+						"lastName": "Fritsche",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "Ulrich",
+						"lastName": "Nieß",
+						"creatorType": "contributor"
+					},
+					{
+						"firstName": "Stadt",
+						"lastName": "Stadt Mannheim",
+						"creatorType": "contributor"
+					},
+					{
+						"firstName": "Johannes",
+						"lastName": "Paulmann",
+						"creatorType": "contributor"
+					},
+					{
+						"firstName": "Ulrich",
+						"lastName": "Nieß",
+						"creatorType": "contributor"
+					},
+					{
+						"firstName": "Susanne",
+						"lastName": "Schlösser",
+						"creatorType": "contributor"
+					}
+				],
+				"date": "2013",
+				"ISBN": "9783897357723",
+				"edition": "2. Aufl",
+				"libraryCatalog": "DNB",
+				"numPages": "960",
+				"place": "Ubstadt-Weiher Heidelberg Neustadt a.d.W. Basel",
+				"publisher": "Verl. Regionalkultur",
+				"series": "Sonderveröffentlichung des Stadtarchivs Mannheim - Institut für Stadtgeschichte",
+				"seriesNumber": "Bd. 39",
+				"attachments": [
+					{
+						"title": "Table of Contents PDF",
+						"mimeType": "application/pdf"
+					},
+					{
+						"title": "Abstract",
+						"mimeType": "text/html"
+					}
+				],
+				"tags": [
+					"(Produktform)Hardback",
+					"(Produktform)Hardback",
+					"(VLB-WN)1558: Hardcover, Softcover / Geschichte/Regionalgeschichte, Ländergeschichte",
+					"(VLB-WN)1558: Hardcover, Softcover / Geschichte/Regionalgeschichte, Ländergeschichte",
+					"Arisierung",
+					"Baden",
+					"Baden",
+					"Geschichte 1933-1969",
+					"Heimatgeschichte",
+					"Heimatgeschichte",
+					"Mannheim",
+					"Nachkriegszeit",
+					"Nachkriegszeit",
+					"Wiedergutmachung",
+					"Zweiter Weltkrieg",
+					"Zweiter Weltkrieg"
+				],
+				"notes": [
+					{
+						"note": "Literaturangaben"
+					}
+				],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://portal.dnb.de/opac.htm?method=showFullRecord&currentResultId=swiRef%3D953552055%26any&currentPosition=1",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "Effiziente Entwurfsverfahren zur hardwarebasierten Signalverarbeitung elementarer Funktionen für die drahtlose Kommunikation / Jochen Rust",
+				"creators": [
+					{
+						"lastName": "Rust",
+						"creatorType": "author",
+						"firstName": "Jochen"
+					}
+				],
+				"date": "2014",
+				"ISBN": "978-3-8440-2706-8",
+				"libraryCatalog": "DNB",
+				"place": "Aachen",
+				"publisher": "Shaker",
+				"attachments": [
+					{},
+					{
+						"title": "Table of Contents PDF",
+						"mimeType": "application/pdf"
+					}
+				],
+				"tags": [],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	}
+]
+/** END TEST CASES **/
```

Comment:
When the translator tries to call the COinS translator this seems sometimes to happen. It is still the same and the error messages begins with
```
         string => TypeError: translate._sandboxManager.sandbox.Array is not a constructor
         stack => Zotero.Translate.Sandbox.Base.loadTranslator/safeTranslator.setHandler/<@chrome://zotero/content/xpcom/translation/translate.js:263:27
```
The line mentioned there is https://github.com/zotero/zotero/blob/4.0/chrome/content/zotero/xpcom/translation/translate.js#L263

Another example is at http://d-nb.info/1027022987